### PR TITLE
Dashboard folder structure (frontend) and dependencies

### PR DESCRIPTION
**Pull request by ogxn**
title

# Issue

# Checklist

- [ ] This PR has been tested
	- [x] Everything still works
	- [x] No major bugs have been made
		- [ ] I have listed minor bugs that have been added
- [x] I have listed all changes that this PR makes
- [ ] This PR needs no further work
- [ ] Manual restart and/or configuration required post-merge
- [ ] README.md or other documentation will be updated by this PR
- [ ] I have listed how to make bot work again post-merge 

- [x] This is a major change
- [ ] This is a minor change
<p/>

- [ ] This is a bug fix
- [ ] This is an enhancement or new feature
- [x] Other: it's folders

### CODE REVIEW

**Comment by TheSuperGamer20578** [comment]
How do you know that there are no new major bugs or that everything still works without testing?

**Comment by ogxn** [comment]
> How do you know that there are no new major bugs or that everything still works without testing?

I didn't change ANYTHING that could affect the bot

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "sudan-bot-frontend",
+    "version": "0.0.1",
+    "description": "Dashboard for TheSuperGamer20578/Sudan-bot",
+    "repository": "https://github.com/TheSuperGamer20578/Sudan-bot/tree/master/frontend",
+    "author": "oworangix",
```
Comment (TheSuperGamer20578) [inline frontend/package.json:6]: Can you be slightly more serious?

**Review by TheSuperGamer20578 — CHANGES_REQUESTED** [review]
I don't know JS stuff but I do know enough to see the one problem.

File: frontend/package.json (hunk 1 above)
Comment (ogxn) [reply frontend/package.json:6]: my npm username is literally oworangix

Comment (TheSuperGamer20578) [reply frontend/package.json:6]: can you send link to npm?

Comment (ogxn) [reply frontend/package.json:6]: https://www.npmjs.com/~oworangix i have...nothing :(

**Review by TheSuperGamer20578 — APPROVED** [review]
Possibly a few things that should change but I don't know JS and I am the only person who can do an approved review :(.

**Comment by ogxn** [comment]
@TheSuperGamer20578 ok in future I will have to verify commits, but can we merge this now?

**Comment by TheSuperGamer20578** [comment]
@ogxn i dont know why this is saying merging blocked what did you do differently to your previous prs?

**Comment by TheSuperGamer20578** [comment]
@ogxn branch protection has been updated i don't know why that happend

**Pull request merged**